subgraph-js/src/federation-atlas.ts

```diff
@@ -0,0 +1,62 @@
+import { Atlas, Schema, gql } from "@apollo/core-schema";
+
+export const SUBGRAPH_BASE = Schema.basic(
+  gql `${'builtin:subgraph-base'}
+    @link(url: "https://specs.apollo.dev/federation/v1.0", as: "",
+          import: "@key @requires @provides @external")
+    @link(url: "https://specs.apollo.dev/tag/v0.1")
+  `)
+
+export const ATLAS = Atlas.fromSchemas(
+  Schema.basic(gql `${'builtin/tag/v0.1'}
+  @id(url: "https://specs.apollo.dev/tag/v0.1")
```

Comment:
We should also add `tag/v0.2` which landed recently (#1652) and will be supported by GA (and it is the version that `federation/v2.0` now use).

Reply:
this directive is the dumbest thing. fixed.